src/ui/FileList/FileList.stories.tsx

```diff
@@ -267,8 +266,13 @@ function Example() {
                       }
                       headNumber={i + 1}
                       baseNumber={i}
-                      headCoverage={coverage[Math.floor(Math.random() * 3) % 3]}
-                      baseCoverage={coverage[Math.floor(Math.random() * 3) % 3]}
+                      headCoverage={
+                        coverage[Math.floor(Math.random() * 3) % 3] ?? null
```

Comment:
won't coverage always be a truthy value here?

Reply:
No, it can be undefined